### PR TITLE
Updated text translations

### PR DESCRIPTION
**Pull request by LambdaLearner**
https://dimagi-dev.atlassian.net/browse/ICDS-1699

### CODE REVIEW

**Review by mkangia — APPROVED** [review]

**Comment by dungeonmaster51** [comment]
> We missed that? I thought we verified the changes yesterday again.

These were the extra translations added, that didn't exist before.

File: custom/icds/templates/icds/messaging/indicators/urd/aww_aggregate_performance.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+آپ کے آنگن واڑی مرکز کیلئے گذشتہ 30 دنوں میں مجموعی کارکردگی کی رپورٹ
+
+1. دوروں کی تعداد / مطلوبہ دوروں کی تعداد:{{ visits }} / 65
```
Comment (akshita-sh) [inline custom/icds/templates/icds/messaging/indicators/urd/aww_aggregate_performance.txt:3]: This is incorrect structure. The{{ visits }}/ 65 should be at the starting as Urdu is read from right to left. Similarly for all the other conditions

Is there a reason why we modified this @dungeonmaster51 ? 

FYI @mkangia

Comment (akshita-sh) [reply custom/icds/templates/icds/messaging/indicators/urd/aww_aggregate_performance.txt:3]: I verified and this happens when we copy paste it. Verified it  on other places in the app. It is written this way only. No action needed on this.

**Review by akshita-sh — APPROVED** [review]
I have verified the changes @mkangia @dungeonmaster51. It looks good. No modifications required.

**Comment by mkangia** [comment]
thank you @akshita-sh

**Pull request merged**